test/plugin_fixtures/vacuum_job.yml

```diff
@@ -0,0 +1,39 @@
+vacuum_job:
+  api_namespace: one
```

Comment:
this definitely needs to be renamed. `one` << someone was lazy with that one and did a copy pasta 🙄